vasculature/vesGraphValidate/vesGraphValidate.m

```diff
@@ -4937,7 +4942,6 @@ function verification_deleteSelectedSegments_Callback(hObject, eventdata, handle
     [ir,~] = find(edgesNew == 0);
     edgesNew(ir,:) = [];
     
-   
     seglstRemove = Data.Graph.segmentstodelete;
     nSegments = size(Data.Graph.segInfo.segEndNodes,1);
     map = (1:nSegments)';
```

Comment:
The logic for creating segMap results in an array with a zero entry. This raises an error on the line "Data.Graph.segInfo.unverifiedIdx = segMap(Data.Graph.segInfo.unverifiedIdx);"